include/openssl/ssl.h

```diff
@@ -1020,6 +1027,7 @@ DECLARE_PEM_rw(SSL_SESSION, SSL_SESSION)
 # define SSL_AD_INTERNAL_ERROR           TLS1_AD_INTERNAL_ERROR
 # define SSL_AD_USER_CANCELLED           TLS1_AD_USER_CANCELLED
 # define SSL_AD_NO_RENEGOTIATION         TLS1_AD_NO_RENEGOTIATION
+# define SSL_AD_END_OF_EARLY_DATA        TLS13_AD_END_OF_EARLY_DATA
```

Comment:
All of the alerts are done this way. I have no idea why. They have a protocol specific version "SSL3_AD_*", "TLS1_AD_*", "TLS13_AD_*" and then a generic name "SSL_AD_*". They are in a public header file so we can't change that. It would be odd to treat this one alert differently.